### DISCUSSION TITLE
IPAM > IP Addresses Attribute Error

### DISCUSSION THREAD

**Discussion by ghost** [netbox-community/netbox#14721] [Help Wanted!]
I had been testing aNetBox on a home setup to see how our use case could be developed and have been more than happy with the product which is a credit to the the developer(s).

Having been suitably impressed I got the go-ahead to create an instance within our workplace and all has gone well, the installation and the data migration to this instance has for the most part worked flawlessly. I ensured I followed the guidelines with respect to the data migration and all is well apart from an error when clicking the IP Addresses.

I get the following error prompt.

![ip_addresses_error](https://github.com/netbox-community/netbox/assets/155900601/ff35a6b0-18b4-42fa-818a-e650b3c020e6)

All other left pane menu links work but this one does not.

Would anyone be able to assist and offer a possible solution to this problem?

Many thanks for considering my request.

**Comment by markkuleinio** [comment]
Please see #11112 if that gets you to resolve the issue.

I also have to say that if you can give detailed information about your data migration, maybe someone could get a hint how the data got messed up in the first place. But it may not be so straightforward after all.

**Comment by ghost** [comment]
Thanks for such a speedy reply.

I took the liberty of just rebooting the VM that I had installed NetBox on and the issue appears to have now resolved itself.

I haven't many IPs mapped at this time but from the post, you provided I understand this error could relate to pointing to a non-existent object.

I will see if I can understand the API side to see if that is possibly the cause here, I'm hopeful that because it is now working it may not be necessary but I of course welcome your thoughts.

**Reply by markkuleinio** [reply]
I haven't personally had reboot fix any data problem in the database, but good if it worked for you, maybe it was something else.

**Comment by ghost** [comment]
Maybe it was beginner's luck but I hope it persists as working.

**Comment by ghost** [comment]
It would appear that during the data migration, some things haven't been fully mapped, I noticed that some of the images in the media folder although present did not show on the respective objects. After the reboot that I mentioned earlier, these have also been correctly associated so perhaps it is best practice to reboot the Netbox instance after a data migration process.
